### ISSUE TITLE
Acceleration not applying to crystal staff

### ISSUE THREAD

**Issue by Saereth** [Lothrazar/RootsClassic#93]
Minecraft version & Mod Version:
mc 1.19.2 
Mod verison: RootsClassic-1.19.2-1.1.35.jar
FTB Skies pack

Single player or Server:
Both

Describe problem (what you were doing / what happened):
The crystal staff should have the acceleration spell (potency 3) added to it when using specific ingredients, the ritual activates properly but end result doesn't show the spell (but shows the staff as containing -something-?).

Steps to reproduce
set up roots altar with 4 brazers around it; mortar and pestle
use the ingredients in the mortar for acceleration with potency 3 ( in this order - oxi daisy, end stone, gold ingot, sugar, dragon eye, 3 glowstone dusts), mix with the pestle
place a newly spawned crystal staff and verdant roots on the altar
place the mixture from the mortar into one of the brazers and light it up
sneak right click on the altar to activate the imbuement process - the animation activates
check the resulting crystal staff

Crash log (if any):
N/A

original issue reported here:
https://github.com/FTBTeam/FTB-Modpack-Issues/issues/3350

**Comment by MoonMagic07** [comment]
I have the same problem @subliminal-bot could you send a picture or describe more in depth on how you did it?

**Comment by Xula1** [comment]
yes @subliminal-bot it would be much appreciated if you could show us how you did it

**Comment by Mathead253** [comment]
@Xula1 @MoonMagic07 You have to place the spell powder into the left corner that is facing the same direction as the staff's handle
![2023-10-10_00 40 53](https://github.com/Lothrazar/RootsClassic/assets/83712695/c86b1c9c-ac9a-4a94-92a7-b3064877e456)

**Comment by Xula1** [comment]
@Mathead253 would it be possible for you to make a video showing it? would really appreciate <3 cause it still doesn't work

**Comment by Mrbysco** [comment]
The issue should be fixed on 1.20.1 and 1.21.1

**Comment by Impos913** [comment]
> The issue should be fixed on 1.20.1 and 1.21.1

On ATM10 0.60 with RootsClassic-1.21.1-1.5.2 tried this ritual 4 times and in 2 it used the powders and didnt add anything to the staff and other 2 times game crashed

 
[crash-2024-09-29_18.22.32-server.txt](https://github.com/user-attachments/files/17179751/crash-2024-09-29_18.22.32-server.txt)

**Comment by Impos913** [comment]
Tried a fifth time and got a different crash this time 
[crash-2024-09-29_18.35.51-server.txt](https://github.com/user-attachments/files/17179805/crash-2024-09-29_18.35.51-server.txt)

**Comment by Mrbysco** [comment]
A new version has been uploaded 1.5.3

**Comment by Impos913** [comment]
ok it worked
